### DISCUSSION TITLE
v0.5.4-alpha Pre Release 5

### DISCUSSION THREAD

**Discussion by ItsRiprod** [ItsRiprod/DeskThing#12] [Announcements]
## ➕ Additions

- :green_square:  Mediawin 'local' temporary view
- :blue_square: Mediawin setting audio output devices **(deferred)**
- :blue_square: Mediawin setting audio input location **(deferred)**
- :blue_square: Spotify setting audio output device **(deferred)**
- :blue_square: Incorporating ADB into the executable
- :blue_square: Finding a way to incorporate the apps into the executable
- :blue_square: Incorporating the webapp into the executable
- :blue_square: Managing the connected devices from the desktop UI
- :blue_square: Discord HUD app **(partially deferred)**
    :blue_square: - View Participants in call
    :blue_square: - Muting **(deferred)**
    :blue_square: - Deafening **(deferred)**
    :blue_square: - Kicking **(deferred)**
    :blue_square: - Mixing **(deferred)**
    :blue_square: - Chat notifications **(deferred)**

## ⚙️ Changes
- Recoded Carthing to use Tailwindcss (mostly)

## 🪲 Bugs
- :green_square: Weather app (all of it basically)
- :green_square: Settings app registering new audio devices
 *Ensure you uninstall all the old apps when you upgrade*
- :green_square: Audio sources array not updating correctly
 *Ensure you uninstall all the old apps when you upgrade*
- :red_square: ADB being ADB and not working very well for MacOS
- :green_square: Car Thing crashing if you purge an app
- :green_square: Car Thing registering unviewable views causing issues with setting preferences
- :green_square: Purging an app doesn't always clear the cache
- :green_square: Spotify will freak out if not given the right API key or secret
- :red_square: Spotify dislikes ads (understandably)
  *Ads will just cause spotify to not work until the spotify ad is over*
- :green_square: Mediawin doesn't save the ID and wait for the app to skip before pulling
- :green_square: Funky error in console regarding foreach when skipping
- :green_square:  Clients Connected now properly shows
- :green_square: Logos everywhere!
- :green_square: Bootup logo!
- :green_square: Minor performance updates
- :green_square: Spotify app crashes when playing a podcast/audiobook
- :green_square: Webapp and desktop app not resizing properly

*:red_square: Unfinished
:orange_square: partially working
:green_square: Added and Implemented
:blue_square: deferred to v0.5.5*

**Full Changelog**: https://github.com/ItsRiprod/DeskThing/compare/v0.5.3-alpha...v0.5.4-alpha

Follow the readme in this release for installation instructions

<hr /><em>This discussion was created from the release <a href='https://github.com/ItsRiprod/DeskThing/releases/tag/v0.5.4-alpha'>v0.5.4-alpha Pre Release 5</a>.</em>

**Comment by OctavXD** [comment]
Can't wait for the widget / android port !
